src/test/test_corpc_version.c

```diff
@@ -157,18 +157,18 @@ test_parse_args(int argc, char **argv)
 
 static void *progress_thread(void *arg)
 {
-	int			rc;
-	crt_context_t		crt_ctx;
+	int			 rc;
+	crt_context_t		 crt_ctx;
+	volatile uint32_t	*shut_down = &test.t_shutdown;
```

Comment:
(style)  Missing a blank line after declarations

(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt